.rubocop.yml

```diff
@@ -33,6 +33,11 @@ Metrics/MethodLength:
 Metrics/ModuleLength:
   Max: 120
 
+# Disabled since it only applies to Rails 5. Reenable when we migrate our apps to Rails 5.
```

Comment:
I upgraded the app to rails 5.0. Everything seems to work, except when I run `rake` I am getting:
```
DEPRECATION WARNING: `redirect_to :back` is deprecated and will be removed from Rails 5.1. Please use `redirect_back(fallback_location: fallback_location)` where `fallback_location` represents the location to use if the request has no HTTP referer information. (called from login at /Users/jgreben/Projects/MyAccount/app/controllers/authentication_controller.rb:7)
```
 Except when I try doing that I get:
```
undefined method `redirect_back' for #<RSpec::ExampleGroups...
```

My bundle update says I am using rspec-core 3.5.4 and rspec-expectations 3.5.0